modules/authorization-manager/src/main/java/org/opencastproject/authorization/xacml/manager/util/AccessInformationUtil.java

```diff
@@ -107,8 +108,9 @@ public static JSONObject serializeManagedAcl(ManagedAcl managedAcl) {
    *           if the <code>acl</code> parameter is null
    */
   public static JSONObject serializePrivilegesByRole(AccessControlList acl) {
-    if (acl == null)
+    if (acl == null) {
       throw new IllegalArgumentException("The parameter trans must not be null");
```

Comment:
where ist the parameter trans here? Is the comment correct here?

Reply:
I just fixed a check style error here since this code was moved and thus wasn't excluded from check style anymore. I'm not responsible for the rest. :D